docs/docusaurus/versioned_docs/version-1.7.0/howtos/inbound_roaming.md

```diff
@@ -50,6 +50,47 @@ subscribers may also work but are not tested yet.
 Configurations with SubscriberDB, local HSS and remote HSS all at the same
 time is not supported yet.
 
+Next diagram shows you a logic diagram of Neutral Host architecture we will
+use on Inbound Roaming to be able to route the request properly. Note that
```

Comment:
`will use` ? Is it not implemented yet? Than it should not be added to the v1.7 documentation.
In case it is already implemented, it should be present tense.